content/marketing/content/editorial/editorial-strategy.md

```diff
@@ -72,24 +72,15 @@ According to a question asked in Slack, our devs would trust the information com
 
 Below is a non-exhaustive, in-progress list of learnings we've gathered from reading the above blogs.
 
-## Balance depth and compactness
+#### Subject matter, style, and tone
 
-The Fly.io blog focuses on two extremes of the length-spectrum, primarily writing either “FYI” posts that take less than five minutes to read or long blog posts that take 20-30 minutes or more.
-
-- Example: [FYI: Phoenix drops webpack and npm for esbuild](https://fly.io/blog/phoenix-moves-to-esbuild-for-assets/) takes 3 minutes to read.
-- Example: [API Tokens: A Tedious Survey](https://fly.io/blog/api-tokens-a-tedious-survey/) takes 39 minutes to read.
-
-The average FYI post takes 3 minutes and the average regular post takes 21.5 minutes.
-
-Another strategy to find this balance is to write multi-part posts. In a series of posts on decision-making at Netflix, the Netflix Engineering blog writes a three minute intro post ([Decision Making at Netflix](https://netflixtechblog.com/decision-making-at-netflix-33065fa06481)) and, as of November 18, with more to come, has written four more posts in this series.
+##### Emphasize hands-on experiences and stories
 
-## Emphasis on hands-on dev experiences and stories
+In the post [How Safari Ruined My Tuesday](https://fly.io/blog/how-safari-ruined-my-tuesday/) on Fly.io, the author hooks the post on a personal story (as exemplified in the title).
 
-In the post [How Safari Ruined My Tuesday on Fly.io](https://fly.io/blog/how-safari-ruined-my-tuesday/), the author hooks the post on a personal story (as exemplified in the title).
+Sentences like “In the whole of the internet, there's only one, lonely Stack Overflow thread talking about this issue affecting mobile Safari” and “This was an annoyingly difficult issue to debug. Safari's web inspector will tell you the computed property of the elements is pointer-events: auto, but it's lying. Elements won't respond to click events. Bad times. It's like living with IE6 again” demonstrate the actual lived experience of software development. In other words, it’s relatable.
 
-Sentences like “In the whole of the internet, there's only one, lonely Stack Overflow thread talking about this issue affecting mobile Safari” and “This was an annoyingly difficult issue to debug. Safari's web inspector will tell you the computed property of the elements is pointer-events: auto, but it's lying. Elements won't respond to click events. Bad times. It's like living with IE6 again” demonstrate the actual lived experience of software development. In other words, it’s relatable. It’s by devs, for devs.
```

Comment:
I've seen negative reactions to "by devs for devs" as it can be perceived as a meaningless marketing term. Even when we know it's true!